cmd/fyne/internal/commands/build.go

```diff
@@ -199,8 +199,8 @@ func (b *Builder) build() error {
 	env := os.Environ()
 
 	if goos == "darwin" {
-		appendEnv(&env, "CGO_CFLAGS", "-mmacosx-version-min=10.13")
-		appendEnv(&env, "CGO_LDFLAGS", "-mmacosx-version-min=10.13")
+		appendEnv(&env, "CGO_CFLAGS", "-mmacosx-version-min=10.11")
```

Comment:
I think this should not go back versions - the new repo has a higher OS requirement and that is OK.